pkg/build/builder/docker.go

```diff
@@ -239,7 +251,12 @@ func (d *DockerBuilder) addBuildParameters(dir string) error {
 	}
 
 	envVars := getBuildEnvVars(d.build)
-	newFileData = appendEnvVars(newFileData, envVars)
+	newFileData = appendMetadata(Env, newFileData, envVars)
+
+	labels := map[string]string{}
+	sourceInfo := d.git.GetInfo(dir)
+	labels = util.GenerateLabelsFromSourceInfo(labels, sourceInfo, DefaultNamespaceLabel)
+	newFileData = appendMetadata(Label, newFileData, labels)
```

Comment:
damn, you 're right, will set the context-dir as part of the `GenerateLabelsFromSourceInfo()`
Thanks for noticing :+1: